### DISCUSSION TITLE
Fill

### DISCUSSION THREAD

**Discussion by sqcred** [Minestom/Minestom#678] [Help] [answered]
Is there a way to fill in multiple blocks at the same time 
like vanilla /fill

**Comment by mythiclight** [comment, marked as answer]
https://wiki.minestom.net/world/batch

**Reply by sqcred** [reply]
I see thanks

**Comment by TheMode** [comment]
Batches can help when reused, but will not bring any performance benefit when only used once.

Palettes have now many specialized methods, which #574 will use (and bring an API for general purpose block placement)
